vmdb/db/migrate/20150414085027_create_container_routes.rb

```diff
@@ -0,0 +1,21 @@
+class CreateContainerRoutes < ActiveRecord::Migration
+  def up
+    create_table :container_routes do |t|
+      t.string     :ems_ref
+      t.string     :name
+      t.timestamp  :creation_timestamp
+      t.string     :resource_version
+      t.string     :namespace
+      t.string     :host
+      t.string     :service_name
```

Comment:
I noticed you mentioned this should be service_id...is the intent to do that in this PR?